force-app/main/sms/classes/HOT_ReminderSMSService.cls

```diff
@@ -301,6 +250,21 @@ public without sharing class HOT_ReminderSMSService {
         return workOrder.Account.CRM_Person__r.INT_KrrMobilePhone__c;
     }
 
+    private static String getRecipientOnThread(Thread__c thread) {
+        if (thread.CRM_Thread_Type__c == 'HOT_BRUKER-FORMIDLER') {
+            return thread.CRM_Account__r.CRM_Person__r.INT_KrrMobilePhone__c;
+        }
+        else if (thread.CRM_Thread_Type__c == 'HOT_TOLK-FORMIDLER') {
+            ServiceResource sr = [SELECT AccountId, HOT_MobilePhone__c FROM ServiceResource WHERE AccountId = :thread.CRM_Account__c AND HOT_MobilePhone__c != null];
```

Comment:
Burde du ikke fjerne `AND HOT_MobilePhone__c != null` og `AND OrdererPhone__c != null`. Du vil vel ha tilbake sr/request uansett, ellers kan du vel få null pointer? Eller returnere null når du ikke finner telefon. Kan vel kanskje løses sånn: `return sr.?HOT_MobilePhone__c;`